### PR TITLE
unbreak build

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh

### CODE REVIEW

File: src/test/resources/projects/clientserver/client/build.gradle [hunk 1]
```diff
@@ -14,4 +14,6 @@ targetCompatibility = 1.7
 dependencies {
   endpointsServer project(path: ":server", configuration: "endpoints")
   compile "com.google.api-client:google-api-client:+"
+  compile "com.google.http-client:google-http-client-gson:1.38.1"
+  compile "com.google.api-client:google-api-client:1.31.2"
```
Comment (chanseokoh) [inline src/test/resources/projects/clientserver/client/build.gradle:17]: I ran the tests locally, and it seems this change is unnecessary.
```suggestion
```

Comment (elharo) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: That's the important part of the change non-locally. It broke immediately for me without this. Strange.

Comment (chanseokoh) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: Can you try again without this? I was able to build remotely without this change. Also it's weird that you needed this change in this `build.gradle` but not in the other `build.gradle` at ` src/test/resources/projects/client` where you replaced `JacksonFactory` as well.

Comment (elharo) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: I'll check but even if it compiles without that it should be there. Relying on transitive dependencies is a maintenance problem. That might be how this broke in the first place.

Comment (chanseokoh) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: Ah, gotcha. In that case, I think it makes sense to declare the dependencies. BTW, we should remove the conflicting one. (The one with `:+`, which conflicts with `:1.31.2`.)

```diff
-  compile "com.google.api-client:google-api-client:+"
   compile "com.google.http-client:google-http-client-gson:1.38.1"
+  compile "com.google.api-client:google-api-client:1.31.2"
```
Also, probably we should make the same change in `build.gradle` at `src/test/resources/projects/client` (in addition to `
src/test/resources/projects/clientserver`)?

https://github.com/GoogleCloudPlatform/endpoints-framework-gradle-plugin/blob/master/src/test/resources/projects/client/build.gradle#L20

Comment (elharo) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: done

Comment (chanseokoh) [reply src/test/resources/projects/clientserver/client/build.gradle:17]: This file too: https://github.com/GoogleCloudPlatform/endpoints-framework-gradle-plugin/blob/gson/src/test/resources/projects/client/build.gradle#L20

**Review by chanseokoh — APPROVED** [review]

**Pull request merged**